### PR TITLE
Address full review feedback

### PR DESCRIPTION
**Pull request by trevordmiller**
# Changes

- Clean up navigation menu icon code suggested by @tayiorbeii 
- Clean up lesson sub-components suggested by @tayiorbeii 
- Clarify promotion in documentation suggested by @ijones16 

# Result For User

No change

---

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,7 +65,9 @@ Routes correspond to screen components. For example, `src/screens/Lessons` is us
...
+
+The idea is that when you create a new file or folder, you put it only where it's needed first (usually deep in the directory tree). Then, as other files need it you "promote" it (move the file or directory) up to the ["lowest common ancestor"](https://en.wikipedia.org/wiki/Lowest_common_ancestor) where it can be imported from multiple places below it. So, for example, when `LessonList` was initially createdd, it was placed in `src/screens/Overview/components/InstructorLessons/components/LessonList`, but eventually, more screens needed `LessonList`, so now it has been moved up to `src/components`. Other generic components that were originally inside individual component directories have now been promoted all the way to the `egghead-ui` project (like `Button`, `Icon`, `Loading`, `Request` etc.).
```
Comment (theianjones) [inline README.md]: 👍 

Small typo 😄 :

Comment (trevordmiller) [reply README.md]: Thanks! Fixed.

**Review by tayiorbeii — APPROVED** [review]
:1st_place_medal:

**Review by theianjones — COMMENTED** [review]
Looks good! -- Didn't refresh, so I didn't see that the pr was merged 😝

**Pull request merged**